client/data/agency-dashboard/use-fetch-dashboard-sites.ts

```diff
@@ -5,14 +5,18 @@ import { wpcomJetpackLicensing as wpcomJpl } from 'calypso/lib/wp';
 import { errorNotice } from 'calypso/state/notices/actions';
 import type { AgencyDashboardFilter } from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/types';
 
-const agencyDashboardFilterToQueryObject = ( filter: AgencyDashboardFilter ) =>
-	filter.issueTypes?.reduce(
-		( previousValue, currentValue ) => ( {
-			...previousValue,
-			[ currentValue ]: true,
-		} ),
-		{}
-	);
+const agencyDashboardFilterToQueryObject = ( filter: AgencyDashboardFilter ) => {
+	return {
+		...filter.issueTypes?.reduce(
+			( previousValue, currentValue ) => ( {
+				...previousValue,
+				[ currentValue ]: true,
+			} ),
+			{}
+		),
+		...( filter.showOnlyFavorites && { show_only_favorites: true } ),
```

Comment:
Why not simply this?

```suggestion
		show_only_favorites: Boolean( filter.showOnlyFavorites ),
```

Reply:
Thanks! Fixed in 09ad234a5ec3a6048bbe2b7471b077ea831c70ac

Reply:
I ended up reverting this one -- it looks like we've done it like this because of the way PHP check for that query parameter works.

Reply:
Aha! got it. Thanks